frontend/gateway/pb/generate.go

```diff
@@ -1,3 +1,3 @@
 package moby_buildkit_v1_frontend
 
-//go:generate protoc -I=. -I=../../../vendor/ --gogo_out=plugins=grpc:. gateway.proto
+//go:generate protoc -I=. -I=../../../vendor/ -I=../../../../../../ --gogo_out=plugins=grpc:. gateway.proto
```

Comment:
follow-up: should do the generation in containers (and validation for CI). Adding more of these includes makes in more dependant on the user environment(eg. symlink in gopath starts to break this)

Reply:
Yes, and I think we should use https://github.com/stevvooe/protobuild